stream_alert/shared/lookup_tables/driver_s3.py

```diff
@@ -195,13 +201,20 @@ def gz_decompress(driver, data):
                 'LookupTable (%s): Data is not compressed; defaulting to original payload',
                 driver.id
             )
-        return data
+        return data.decode()
```

Comment:
FYI; this is a problem that exists on live right now due to `zlib` on Python3 not accepting strings. It only accepts Bytes-like.